Source/DafnyCore/DafnyFile.cs

```diff
@@ -16,13 +18,30 @@ public class DafnyFile {
   public string BaseName { get; private set; }
   public bool IsPreverified { get; set; }
   public bool IsPrecompiled { get; set; }
-  public bool IsPrerefined { get; private set; }
```

Comment:
Off topic but when did this appear and what was it used for?

Reply:
It was needed when .doo files still contained resolved code instead of just parsed code.